### DISCUSSION TITLE
Creating an Infrastructure Cloud project

### DISCUSSION THREAD

**Discussion by Johannes-lab** [iTwin/community#61] [Q&A] [answered]
Hello, I am looking for a way to create an iTwin, viewable at infrastructurecloud.bentley.com (BIC) via the create iTwin API. Specifying; 
 
       "class": "Endeavor",
        "subClass": "Project",
        "type": "Construction",
        
creates a project under Connect only.
Whereas

        "class": "Thing",
        "subClass": "Asset",
        "type": "Construction",

will create an iTwin Experience project.
        
What is the class and subclass for BIC? I know Synchro Control projects require provisioning through something like https://github.com/BentleySystems/ES-API-Samples/tree/main/EsApiProjectsSampleApp

would this be a similar requirement for BIC?

Thanks.

**Comment by davidhjones** [comment, marked as answer]
Yes, IC will not surface iTwins created directly via the iTwin Platform API. The product requires that you take the appropriate steps (via the provisioning service) to create the iTwin, and only then will those iTwins surface in IC. 

@anirban-chakraborty-bentley

**Reply by Johannes-lab** [reply]
Thanks for your reply. That provisioning service, is that accessible somewhere and meant to be used by end users?
